.github/workflows/build_and_test.yml

```diff
@@ -71,17 +73,37 @@ jobs:
              export IPDK_RECIPE=`pwd`
              cd $IPDK_RECIPE/setup
              cmake -B build -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/deps
-             cmake --build build -j6
+             cmake --build build -j6   
 
+      - name: 'Run Stratum unit tests'
+        run: |
+             cd /tmp
+             curl -o bazel-4.2.3-installer-linux-x86_64.sh https://github.com/bazelbuild/bazel/releases/download/4.2.3/bazel-4.2.3-installer-linux-x86_64.sh
+             chmod +x bazel-4.2.3-installer-linux-x86_64.sh
+             ./bazel-4.2.3-installer-linux-x86_64.sh --user
+             export PATH="$PATH:$HOME/bin"
+             cd ${GITHUB_WORKSPACE}/ipdk.recipe/stratum/stratum
+             export DPDK_INSTALL=${GITHUB_WORKSPACE}/install
+             echo "DPDK" > ${GITHUB_WORKSPACE}/install/share/TARGET
+             xargs -a ${GITHUB_WORKSPACE}/ipdk.recipe/.github/dpdk-tests.txt bazel test --define target=dpdk
+             cd bazel-testlogs/stratum
+             zip stratum_ut_logs.zip `find . -name test.xml -print`
+                          
+      - name: 'Upload stratum-ut-result artifact'
+        uses: actions/upload-artifact@v3
+        with:
+          name: stratum-ut-result
+          path: ipdk.recipe/stratum/stratum/bazel-testlogs/stratum/stratum_ut_logs.zip
```

Comment:
How do we access these logs? Do they need to get versioned to differentiate between subsequent runs?

Reply:
I believe the intention is to feed the test results to a dashboard. I don't know if that includes providing access to the individual test logs.

Reply:
Yes, for now, we need some modifications to display in dashboard, but test results are avilable under artifacts section of each GitHub run

![image](https://github.com/ipdk-io/networking-recipe/assets/71247399/15a5d0e2-1808-4e59-9b7d-6823d39826d6)